graylog2-web-interface/src/components/streamrules/StreamRulesEditor.jsx

```diff
@@ -46,7 +46,13 @@ class StreamRulesEditor extends React.Component {
     index: '',
   }
 
-  state = {};
+  constructor() {
+    super();
```

Comment:
I think `constructor` needs to pass `props` to `super` or else it could lead to [bugs](https://reactjs.org/docs/react-component.html#constructor)
```suggestion
  constructor(props) {
    super(props);
```